src/templates/dso_api/dynamic_api/docs/rest/base.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>{% load static %}
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Amsterdam DataPunt API Documentatie</title>
+  <link rel="stylesheet" type="text/css" href="{% static 'rest_framework/css/bootstrap.min.css' %}"/>
+  <link rel="stylesheet" type="text/css" href="{% static 'rest_framework/css/bootstrap-tweaks.css' %}"/>
+  <link rel="stylesheet" type="text/css" href="{% static 'rest_framework/css/default.css' %}"/>
+  <link rel="stylesheet" type="text/css" href="{% static 'dso_api/dynamic_api/css/docs.css' %}"/>
+</head>
+<body>
+  <div class="container">
+
+<h2>Introductie API Keys</h2>
+<blockquote style="border-color: #ec0101">
+    <p>
+        Het Dataplatform van de gemeente Amsterdam gaat het gebruik van een identificatie key
+        bij het aanroepen van haar API's vanaf 1 februari 2024 verplicht stellen.
```

Comment:
Is deze info/datum nog up-to-date? De hoofdpagina van de dso api noemt geen datum